src/docbuilder/rustwide_builder.rs

```diff
@@ -166,6 +166,9 @@ impl RustwideBuilder {
         for target in &targets_to_install {
             self.toolchain.add_target(&self.workspace, target)?;
         }
+        // NOTE: rustup will automatically refuse to update the toolchain
+        // if `rustfmt` is not available in the newer version
```

Comment:
Do we _want_ this behavior, or should we not block the update on it and only have optional `rustfmt` support? This will currently result in occasionally having like a week without updating the nightly.

Reply:
I think going a week without updating is better than having rustfmt support you can only sometimes depend on. See https://github.com/rust-lang/docs.rs/issues/357#issuecomment-606252725.